lib/OpenQA/Worker/Common.pm

```diff
@@ -336,31 +340,35 @@ sub call_websocket {
         $ua_url => {'Sec-WebSocket-Extensions' => 'permessage-deflate'} => sub {
             my ($ua, $tx) = @_;
             if ($tx->is_websocket) {
-                # keep websocket connection busy
-                $tx->send({json => {type => 'ok'}});    # Send keepalive immediately
-                $hosts->{$host}{timers}{keepalive}
-                  = add_timer("keepalive-$host", 10, sub { $tx->send({json => {type => 'ok'}}); });
-
-                $hosts->{$host}{timers}{status} = add_timer(
-                    "workerstatus-$host",
-                    15,
-                    sub {
-                        log_debug("Sending worker status to $host");
-                        $tx->send(
-                            {
-                                json => {
-                                    type => 'worker_status',
-                                    (status => 'working', job => $job) x !!($job),
-                                    (status => 'free') x !!(!$job),
-                                }});
-                    });
+
+                my $send_status = sub {
+                    log_debug("Sending worker status to $host");
+                    $tx->send(
+                        {
+                            json => {
+                                type => 'worker_status',
+                                (status => 'working', job => $job) x !!($job),
+                                (status => 'free') x !!(!$job),
+                            }});
+                };
+
+                my $i = looks_like_number($instance) ? $instance : 1;
+            # Feature scaling - we do not know the instance maximum per worker, we take DEAD_DETECTION_THRESHOLD as max.
+                my $imax = $i > DEAD_DETECTION_THRESHOLD ? $i : DEAD_DETECTION_THRESHOLD;
+                my $worker_status_timer
+                  = int(MIN_STATUS_TIMER + ((($i - 1) * (DEAD_DETECTION_THRESHOLD - MIN_STATUS_TIMER)) / ($imax - 1)));
```

Comment:
Right, and that would add to the code *just another* useless variable for **1** statement forcing even the logic to come up with a variable name, that would be *meaningless* anyway:
```perl
my $foo = ($i - 1) * (DEAD_DETECTION_THRESHOLD - MIN_STATUS_TIMER);
my $bar = ( $foo / ($imax - 1));
my $worker_status_timer = int(MIN_STATUS_TIMER + $bar);
```
How you are going to call `$foo` and `$bar` then? I don't see a good reason to do that
At this point, if the line break is the problem, it's better to just shrink the constant name, e.g:
```perl
my $status_timer = int(MIN_TIMER + ((($i - 1) * (MAX_TIMER - MIN_TIMER)) / ($imax - 1)));
```